### PR TITLE
finished NewsPage

### PR DESCRIPTION
**Pull request by Lolimkeri**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/AdditionalPages/NewsPage/BreadCrumb/BreadCrumb.component.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+const BreadCrumb = ({ separator = '->', news }: Props) => (
+    <Breadcrumb className="breadcrumbContainer" separator={separator}>
+        <Breadcrumb.Item className="activeLink" href="_blank">
+            Головна
+        </Breadcrumb.Item>
+        <Breadcrumb.Item href="_blank">
+            Новина
+        </Breadcrumb.Item>
```
Comment (NadiaKishchuk) [inline src/features/AdditionalPages/NewsPage/BreadCrumb/BreadCrumb.component.tsx:19]: please, set correct href here

File: src/features/AdditionalPages/NewsPage/News.component.tsx [hunk 2]
```diff
@@ -2,82 +2,141 @@ import './News.styles.scss';
...
     const newsUrl = useRouteUrl();
-    const {newsStore, imagesStore} = useMobx();
-    const { setCurrentNewsId } = newsStore;
-    console.log(newsUrl);
+    const [newsImg, setNewsImg] = useState<HTMLElement>(); 
+    const { newsStore, imagesStore } = useMobx();
     var { value } = useAsync(() => NewsApi.getByUrl(newsUrl), [newsUrl]);
     const news = value as News;
-    const id = news?.id;
+    newsStore.fetchNewsAll();
```
Comment (NadiaKishchuk) [inline src/features/AdditionalPages/NewsPage/News.component.tsx:22]: set it in useEffect hook

File: src/features/AdditionalPages/NewsPage/News.styles.scss [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                        
+                    }
+                }
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 400px) {
+    .newsContainer{
```
Comment (NadiaKishchuk) [inline src/features/AdditionalPages/NewsPage/News.styles.scss:257]: we start the mobile version from 480 if I don`t mistake

**Review by NadiaKishchuk — CHANGES_REQUESTED** [review]

File: src/app/router/Routes.tsx [hunk 4]
```diff
@@ -86,6 +87,7 @@ const router = createBrowserRouter(createRoutesFromElements(
         <Route path={FRONTEND_ROUTES.OTHER_PAGES.PARTNERS} element={<PartnersPage />} />
         <Route path={FRONTEND_ROUTES.OTHER_PAGES.SUPPORT_US} element={<SupportUs />} />
         <Route index path="/:id" element={<StreetcodeContent />} />
+        <Route index path="news/:id" element={<NewsPage />} />
```
Comment (Tysyatsky) [inline src/app/router/Routes.tsx:90]: add 'news' constant to the routes

**Review by ormykhalyshyn — APPROVED** [review]

**Pull request merged**